src/components/TodoList.jsx

```diff
@@ -1,14 +1,16 @@
 import { useSelector, useDispatch } from "react-redux";
-import { deleteTodo, completedTodo } from "../redux/action/actions";
+import { completedTodo, deleteTodo } from "../redux/todosSlice";
```

Comment:
Use consistent naming, since they are actions they should be named as verbs like ` completeTodo, deleteTodo`

Reply:
fixed